tests/contrib/aiohttp/conftest.py

```diff
@@ -24,7 +24,7 @@ def app_tracer(tracer, loop):
 
     @pytest.fixture
     def patched_app_tracer(app_tracer):
-        patch()
+        patch_jinja2()
```

Comment:
why this change? this should still work as-is right?

we probably want to continue to verify that `patch()` does patch `aiohttp_jinja2` ?

Reply:
The server instrumentation tests use an aiohttp client and so by using `patch` here we end up getting client spans in the tests for the server.

Reply:
Might be worth leaving a comment here though, so I'll do that since it isn't obvious

Reply:
Actually opting to not do this since I think it is pretty obvious from reading the code. I tried adding a comment but it seemed out of place.